docs/reference/master-commandline-reference.md

```diff
@@ -34,7 +34,7 @@ Print version and exit.
 
 The `-feature-gates` flag is used to enable or disable non GA features.
 The list of available feature gates can be found in the [feature gates
-documentation](../feature-gates.md).
+documentation](feature-gates.md).
```

Comment:
Looking at the netlify preview it didn't work (perhaps some bug in some 3rd party component in our docs building pipeline) :thinking: Maybe the links should not be split accross multiple lines(?)

```suggestion
[feature gates documentation](feature-gates.md).
```

Reply:
> Looking at the netlify preview it didn't work (perhaps some bug in some 3rd party component in our docs building pipeline) 🤔 Maybe the links should not be split accross multiple lines(?)

will check to see how to address it

Reply:
I just found why it doesn't work in netlify preview. 

In netlify preview, if you click the item in the navigation bar "10.Feature Gates", it's pointing to "https://deploy-preview-1821--kubernetes-sigs-nfd.netlify.app/reference/feature-gates", and it works there. There is no ".md" for the page URL.

But in the "master-commandline-reference" page, the link to Feature Gates including suffix ".md" in the URL, that means URL "https://deploy-preview-1821--kubernetes-sigs-nfd.netlify.app/reference/feature-gates.md" won't work.

Need to figure out how to address the issue.

Reply:
It should change the .md -> .html automatically when building the site. Do what I suggested above^ and I think it fixes it

Reply:
yes, it's the multi-line issue, and some bug in jekyll rendering.

Both .md link and html link work Now.